changelogs/unreleased/gh-8146-space-and-index-names-in-iproto-requests.md

```diff
@@ -0,0 +1,10 @@
+## feature/box
+
+* Added support for accepting IPROTO requests with space or index name instead
```

Comment:
```
with space -> 
with a space
```

Reply:
I guess, I should also change `instead of identifier -> instead of an identifier`. Fixed.